hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CloneSnapshotProcedure.java

```diff
@@ -521,4 +526,11 @@ private void addRegionsToMeta(final MasterProcedureEnv env) throws IOException {
     metaChanges.updateMetaParentRegions(env.getMasterServices().getConnection(), newRegions);
   }
 
+  /**
+   * Exposed for Testing: HBASE-26462
+   */
+  public boolean getRestoreAcl() {
```

Comment:
Please add a RestrictedApi annotation to limit the usage only in tests.

This is an example:

https://github.com/apache/hbase/blob/dcd622d702c9300d81f66756308e4dbe801b1acf/hbase-client/src/main/java/org/apache/hadoop/hbase/client/AbstractRpcBasedConnectionRegistry.java#L224